3_run_rsync/README.md

```diff
@@ -27,8 +27,19 @@ mig_dest_ssh_private_key: "" # path to private key used for SSH auth into transf
 
 # Wait for transfer pod service ELB deletion to complete before proceeding to next PVC
 wait_for_finalizer: false
+
+dynamicProvisionerName = "kubernetes.io/glusterfs"
 ```
 
+Replace the dynamicProvisionerName by picking one from the supported options
+in [run-rsync.yml.example](./vars/run-rsync.yml.example).
+
+_*Note:*_ pvc-migrate assumes that all the pvc's in your namespaces are
+provisioned by a single provisioner. Please take a look at open issues to
+learn more about the problem.
```

Comment:
Should link to the specific issue here and categorize as "known limitation".

Reply:
I didnt link specific issue because the document will not age well. If we do decide to close the existing issue create a new one, this document will age very badly. If you insist, I can add it though

Reply:
IMO if the issue is closed and someone follows the link, they'll find out its been fixed :)